packages/reactivity/src/effect.ts

```diff
@@ -88,7 +88,7 @@ function run(effect: ReactiveEffect, fn: Function, args: any[]): any {
   if (!effect.active) {
     return fn(...args)
   }
-  if (activeReactiveEffectStack.indexOf(effect) === -1) {
+  if (!activeReactiveEffectStack.includes(effect)) {
```

Comment:
According to existing benchmarks, in latest versions of Chrome these two approaches are pretty close on performance. Shall we prefer `includes` here? If we never use `includes` in our code base we potentially reduces the need to polyfill `Array.prototype.includes` for future compatible builds (thus may result in smaller size overhead) but it migh be too trivial. WDYT? @yyx990803

Reply:
emmm,I don't know the performance difference between `indexOf` and `includes`，but there are other places used `includes`

```javascript
// src/scheduler.ts
export function queueJob(job: () => void) {
  if (!queue.includes(job)) {
    queue.push(job)
    if (!isFlushing) {
      nextTick(flushJobs)
    }
  }
}
```

Reply:
My point is if we prefer `indexOf` over `includes` for polyfill reasons we should eliminate all usage of `includes` in our codebase. Otherwise we should always prefer `includes` because performance is pretty close.

Reply:
Yes let's use `includes` everywhere since it's more readable and semantically correct. The polyfill is pretty light too. The general idea is we will use modern features as long as it's an objective improvement over ES5 counterparts and not too costly to polyfill in IE11.